uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/EntityPersonAttributesGroupStore.java

```diff
@@ -317,7 +317,7 @@ private GroupDefinition initGroupDef(IPersonAttributesGroupDefinition group) {
             TestGroup tg = new TestGroup();
             Set<IPersonAttributesGroupTestDefinition> tests = testGroup.getTests();
             for(IPersonAttributesGroupTestDefinition test : tests) {
-                IPersonTester testerInst = initializeTester(test.getTesterClassName(), test.getAttributeName(), test.getTestValue());
+                IPersonTester testerInst = initializeTester(test);
                 tg.addTest(testerInst);
```

Comment:
Okay @apetro & @timlevett  -- production outage may be too severe here.

But I still feel that the group should not be in service.  In fact the "worst case" -- in this case -- would be that suddenly you *are* a member of a group you shouldn't be.  Test groups are "or-ed" together, but tests are "and-ed" together.  If a test drops out of a test group that had multiple tests, the membership gets wider (not narrower).